v2/thirdparty/serverless/with-aws-lambda/authorizer.mdx

```diff
@@ -20,7 +23,124 @@ Follow [step 1](./frontend) and [step 2](./backend-config).
 ## 2) Create a new file `index.js`
 Use the code below as the handler for the lambda. Remember that whenever we want to use any functions from the `supertokens-node` lib, we have to call the `supertokens.init` function at the top of that serverless function file. We can then use `getSession()` to get the session.
```

Comment:
This section should go inside node js tab, and a similar section should be added for python tab

Reply:
Done.